### DISCUSSION TITLE
Typescript errors

### DISCUSSION THREAD

**Discussion by Mw3y** [prisma/prisma#12840] [Q&A] [answered]
Hello,

I'm getting many type errors in this code. How can I fix that? (I don't know where you can find the list of types).

![image](https://user-images.githubusercontent.com/34509813/163605321-40bdc2ac-e10a-4e5a-858f-55ee79d56043.png)

**Comment by austincrim** [comment, marked as answer]
Hey @Mw3y 👋 ,

You are getting these errors because of how you are creating the `prisma` variable. The `prisma` variable is not getting the correct type because of the `global.prisma ||` before the assignment.

It is probably possible to workaround the types, but it is probably easiest to have TypeScript ignore those lines.

**Reply by Mw3y** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
